lib/hydra/pcdm/models/concerns/object_behavior.rb

```diff
@@ -4,9 +4,14 @@ module Hydra::PCDM
   module ObjectBehavior
     extend ActiveSupport::Concern
 
+    class Proxy < ActiveFedora::Base
```

Comment:
1. Should this be nested like this? Maybe this should go in Aggregates? Or in the PCDM namespace?
2. Should we define the other properties? IE proxyIn
3. Can we use the ORE vocab?